### ISSUE TITLE
Attribute auto-generate ID script implementation

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-geospatial#17]
Work with Mark Hawkins from CTM on an auto-generate ID script implementation for GIS layers.

**Comment by jaime-mckeown** [comment]
Mark Hawkins provided me with a good GitHub Wiki that breaks down the options for auto-generating IDs. We spoke about Oracle Triggers, he setup a test layer for local_area_traffic_management in GISMAINTdev with the Oracle Trigger setup so I can test it.

**Comment by jaime-mckeown** [comment]
Tested the FC that Mark setup for me in GISMAINTdev, seems to work fine, updates the unique ID field with each record creation. Simple, does what it needs to do, no interaction required of editors which will keep this field duplicate free.

**Comment by jaime-mckeown** [comment]
Mark at CTM is going to show me how to setup my own Oracle Trigger on GISMAINTdev so I can test this out with the new parking inventory data that has relationship classes. This will be a good test for this option.

**Comment by jaime-mckeown** [comment]
Worked on setting up the trigger and sequence in Oracle SQL Developer. The roles weren't already created, Mark put in a request to the DBAs to create a new role on GISMAINTTEST so I can work with triggers and sequence on Test.

**Comment by jaime-mckeown** [comment]
Mark got the roles setup. I ended up asking him to get this setup for me as I don't have extra time to try and figure out how this works. He offered to setup and configure these for us, as it was not necessary for us to do it ourselves. After he gets the LATM data setup in GISMAINT1, I will work with him to get it setup on the parking inventory test data.

**Comment by jaime-mckeown** [comment]
Waiting for LATM to go through change management board. Once that happens, Mark will be moving to GISMAINT with Oracle Trigger working.

**Comment by jaime-mckeown** [comment]
Phone meeting with Bruce Bacia at CTM today to discuss Oracle Trigger. Mark is out and my request has been reassigned to Bruce. I filled him in on what Mark and I were doing. We did a couple screen shares to see how the layer was coding records. He asked me to share the GISMAINTtest atd_admin connection with him and to have another user test the layer to see how it performs. Will be in touch again to move forward.
